### PR TITLE
Auto preset creation modal

### PR DESCRIPTION
**Pull request by Kingdorian**
This adds a modal which can be used to use the auto preset functionality. The implementation of this had to be altered a fair bit to be able to draw the rectangles on the image in the modal.

### CODE REVIEW

File: backend/src/main/java/com/benine/backend/http/PresetsHandler.java [hunk 1]
```diff
@@ -63,7 +65,7 @@ public void handle(String s, Request request, HttpServletRequest req, HttpServle
    */
   private String getRoute(Request request) {
     String path = request.getPathInfo();
-
+    System.out.println(path);
```
Comment (bvanwijk) [inline backend/src/main/java/com/benine/backend/http/PresetsHandler.java]: I think this can be removed

Comment (Kingdorian) [inline backend/src/main/java/com/benine/backend/http/PresetsHandler.java]: Fixed in d3e9588.

**Comment by bvanwijk** [comment]
I think it looks great, I agree with merging.

**Comment by egedebruin** [comment]
Great work, I agree with merging too.

**Comment by jochem725** [comment]
Looks like a great addition. The view is really awesome.

**Pull request merged**